site/es/blog/new-in-devtools-95/index.md

```diff
@@ -103,54 +124,65 @@ Chromium issues: [1076820](https://crbug.com/1076820), [1119900](https://crbug.c
 
 
 <!-- ## Lighthouse 8.4 in the Lighthouse panel {: #lighthouse } -->
+## Lighthouse 8.4 en el panel Lighthouse {: #lighthouse }
 
 <!-- The **Lighthouse** panel is now running Lighthouse 8.4. Lighthouse will now detect if the [Largest Containful Paint (LCP)](https://web.dev/lcp) element was a lazy-loaded image and recommend removing the `loading` attribute from it. -->
 
+El panel **Lighthouse** está usando Lighthouse 8.4. Lighthouse detectará ahora si el elemento [Largest Containful Paint (LCP)](https://web.dev/lcp) era una imagen con carga diferida y recomendará eliminar el atributo `loading` de ella.
+
 <!-- Check out the [What’s new in Lighthouse 8.4](/blog/lighthouse-8-4/) for more details on the updates. -->
+Revisa el [Novedades de Lighthouse 8.4](/blog/lighthouse-8-4/) para más detalles sobre los cambios.
 
-{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="The lazy-loaded LCP audit in a Lighthouse report", width="800", height="502", class="screenshot" %}
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="La auditoría del LCP con carga diferida en el reporte de Lighthouse", width="800", height="502", class="screenshot" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/649a979e4de2cf38430e46e7198b11ba8a830388 #}
 
 Chromium issue: [772558](https://crbug.com/772558)
 
-
 <!-- ## Sort snippets in the Sources panel {: #snippets } -->
+## Ordena los fragmentos en el panel Fuentes {: #snippets }
 
-<!-- The [snippets](/docs/devtools/javascript/snippets/) in the **Snippets** pane under the **Sources** panel are now sorted alphabetically. Previously, it’s not sorted. -->
+Los [fragmentos de código](/docs/devtools/javascript/snippets/) en la pestaña **Fragmentos** bajo el panel **Fuentes** están ordenados ahora alfabéticamente. Antes no lo estaba.
 
-<!-- Utilize the snippets feature to run commands quicker. Watch this video for a [tip](https://youtu.be/NOal2gTzftI?t=176)! -->
+Utiliza la característica de fragmentos para ejecutar comandos más rápido. ¡Mira este vídeo para una [sugerencia](https://youtu.be/NOal2gTzftI?t=176)!
```

Comment:
In Spanish there is a large community of developers who do not translate the term "**snippets**" and can quickly recognize it.

"Utiliza la característica snippets/fragmentos de código para ejecutar comandos más rápido. ¡Mira este vídeo para ver un ejemplo!"

I know that "tip" is not "un ejemplo", but in Spanish it sounds weird. I'll leave you a "ejemplo".

Reply:
In this case I prefer to stick to DevTools official translation of **snippets** as it would be weird to use _snippets_ here but users finding the term translated on the app.

<img width="573" alt="CleanShot 2021-09-28 at 09 17 48@2x" src="https://user-images.githubusercontent.com/1561955/135041169-31aed7cc-298e-439d-bf34-612c311b92ed.png">

For me, _sugerencias_ sounded good but I'm changing for your suggestion! 👍